INFRASTRUCTURE.md

```diff
@@ -0,0 +1,77 @@
+# Our Infrastructure
+
+First, let's address some details of this document.
+
+<details>
+  <summary>What, why, who...</summary>
+
+  ## What?
+
+  This document outlines the various systems that JSON Schema the organisation uses and maintains.
+  This does not include social media profiles.
+
+  ## Why?
+
+  It's part of the required tasks to be completed by the OpenJS Foundation, forming an item of the [onboarding checklist](https://github.com/openjs-foundation/cross-project-council/blob/main/PROJECT_PROGRESSION.md#onboaghrding-checklist).
+  It's also probably good practice.
+
+  This enables members of the JSON Schema team, and the OpenJS Foundation, to have oversight and management of such infrastructure should it be required.
+
+  ## Who?
+
+  This was mainly a list created by Ben Hutton (@relequestual).
+  Others are free to amend and update this (Via Pull Requests) as is required.
```

Comment:
```suggestion
  Others are free to amend and update this (via Pull Requests) as is required.
```

You can leave PRs capitalized if you want, but it's not necessary.

Reply:
I tend to capitalize actions or things related to GitHub. Granted here it's not needed.
I'm not looking for this document to be gramatically perfect, but just to have the right information included 😅